src/components/Navbar/DropdownMenu.tsx

```diff
@@ -40,7 +38,7 @@ export function DropdownMenu({ links }: DropdownMenuProps) {
               key={target}
               target={target}
               onClick={() => setIsOpen(false)}
-              currentPath={pathname}
+              currentPath={"/"}
```

Comment:
no just lazy lol

Reply:
I don't know a straightforward way to get the current path on a server component with the app router nextjs - the OG implementation was using react router.  

this isn't an issue because it's a single page webapp with a fake navigation that just links back to another domain.